xonsh/pyghooks.py

```diff
@@ -317,12 +317,12 @@ def style_name(self):
     def style_name(self, value):
         if self._style_name == value:
             return
-        if value in STYLES:
-            cmap = STYLES[value]
-        else:
+        if value not in STYLES:
             warn('Could not find style {0!r}, using default'.format(value),
                  RuntimeWarning)
-            cmap = DEFAULT_STYLE
+            value = 'default'
```

Comment:
IMO `value = self._style_name or 'default'` would lead to better UX ... but that's an awful lot of indirection code-wise ...